api/api.go

```diff
@@ -121,7 +121,7 @@ func Mount(
 		err = fmt.Errorf("Mount: initialization failed")
 		return
 	}
-	server := fuseutil.NewFileSystemServer(FusePanicLogger{fs})
+	server := fuseutil.NewFileSystemServer(FusePanicLogger{Fs: fs})
```

Comment:
Is this needed?

Reply:
no, it's not.

I just got a warning `struct literal uses unkeyed fields` in my IDE and though we should address it.

Reply:
I'm happy to revert this file if that's a better choice